.github/workflows/release.yml

```diff
@@ -107,6 +107,9 @@ jobs:
           cargo build --release --target=${{ matrix.target }}
       - name: Copy
         run: cp target/${{ matrix.target }}/release/libviam_rust_utils.dylib builds/libviam_rust_utils-${{ matrix.platform }}.dylib
+      - name: Correct install path
+        run: |
+          install_name_tool -id "@rpath/libviam_rust_utils-${{ matrix.platform }}.dylib" builds/libviam_rust_utils-${{ matrix.platform }}.dylib
```

Comment:
Two things here:

- Setting the install name to `@rpath/libviam_rust_utils-${{ matrix.platform }}.dylib` means that the file must be on disk with the `-{{matrix.platform}}` component on the name. That isn't the current practice in the C++ SDK, where the file is installed as `libviam_rust_utils.dylib`.

- I don't think `@rpath` is entirely sufficient. There is nothing in the C++ SDK build that will put `$PREFIX/lib` or `../lib` into the rpath stack, which would be required for this to resolve. Now, maybe the C++ SDK could (should?) do that in order to make it work.

Reply:
Good catch, thanks! Fixed the filename issue. As to whether `@rpath` is sufficient, we can work on fixing that in the SDK?